src/standalone/Selector/BaseSelector.tsx

```diff
@@ -243,10 +263,16 @@ const BaseSelector = <DataT extends BaseSelectorData>(
 		noGroupLabel,
 		disableGroupSorting,
 		groupSorter,
+		switchLabel,
 	} = props;
 	const classes = useThemeStyles(
 		(props as unknown) as BaseSelectorProps<BaseSelectorData>
 	);
+	const customClassesSwitch = useStyles();
```

Comment:
any reason this is not part of the main styles?

```tsx
const customClassesSwitch = useMemo(() => ({
  switch: classes.switchStyles
}), [classes.switchStyles]);
```

Reply:
@TQjona , It's just UI adjust for switch to fit in the layout properly for selector. So, I don't think it's consider as a main style.